backend/src/controllers/recipe.ts

```diff
@@ -13,4 +14,15 @@ export default {
       next(error);
     }
   },
+  search: async (req: Request, res: Response, next: NextFunction) => {
+    try {
+      checkQueryParams(['title', 'description'], req.query);
+      const { title, description } = req.query as { title?: string; description?: string; };
+      const recipes = await recipeSerice.search({ title: title || '', description: description || '' });
```

Comment:
```suggestion
      const recipes = await recipeSerice.search({ title, description, });
```

fais passer cette logique dans le service